packages/lfrqa/src/aviary/envs/lfrqa/env.py

```diff
@@ -118,102 +120,129 @@
 )
 
 
-class LFRQAPairwiseEvalEnv(GradablePaperQAEnvironment[dict]):
-    """Environment to evaluate paperqa's vs human's answers on Long Form RAG QA questions."""
+class LFRQAQuestion(MultipleChoiceQuestion):
+    gt_doc_ids: list[int]
+    grading_rewards: dict[str, float] = Field(
+        default_factory=lambda: {"win": 1.0, "tie": 0.0, "lose": -1.0}
+    )
 
-    def __init__(
-        self,
-        *args,
-        qid: str | UUID,
-        question: str,
-        human_answer: str,
-        gt_doc_ids: list[int],
-        pairwise_eval_llm: LLMModel | str = CommonLLMNames.GPT_4O.value,
-        **kwargs,
-    ):
-        kwargs["query"] = question
-        kwargs["docs"] = Docs()
-        super().__init__(*args, **kwargs)
+    @model_validator(mode="before")
+    @classmethod
+    def _validate_fields(cls, data: Mapping) -> dict:
+        data = deepcopy(data)  # Avoid mutating input
```

Comment:
Instead of `deepcopy`, can we just make a new `dict` and place values in it:

```python
processed_data = {"options": [], "prompt_without_options": True}
for k, v in data.items():
    if k == "answer":
        processed_data["ideal_answer"] = data["answer"]
    if k == "qid":
        processed_data["question_id"] = data["qid"]
    ...
```